test/ote/utils.go

```
@@ -0,0 +1,319 @@
+	pods, err := clientset.CoreV1().Pods(ns).List(ctx, metav1.ListOptions{LabelSelector: label})
+	if err != nil {
+		return nil, err
+	}
+	var names []string
+	for _, p := range pods.Items {
+		names = append(names, p.Name)
+	}
+	return names, nil
+}
+
+func getOVNKMasterPod(ctx context.Context, clientset *kubernetes.Clientset) (string, error) {
```

golangci-lint (2.12.2)

[error] func getOVNKMasterPod is unused

(unused)